src/cljs/ataru/virkailija/application/attachments/liitepyynto_information_request_handlers.cljs

```diff
@@ -0,0 +1,261 @@
+                   "/field-deadline")
+              {:response-format (ajax/ring-response-format
+                                 {:format (ajax/json-response-format {:keywords? true})})
+               :handler         (fn [response]
+                                  (on-success
+                                   (map (fn [deadline]
+                                          (let [datetime (->> (:deadline deadline)
+                                                              (f/parse iso-formatter)
+                                                              c/to-default-time-zone)]
+                                            [(keyword (:field-id deadline))
+                                             (f/unparse date-formatter datetime)
+                                             (f/unparse time-formatter datetime)]))
+                                        (:body response))
+                                   (get-in response [:headers "last-modified"])))
+               :error-handler   (fn [_] (on-error))
+               :headers         {"Caller-Id" (aget js/config "virkailija-caller-id")}})))
+
+(re-frame/reg-fx
+  :liitepyynto-information-request/get-deadline
+  (fn [{:keys [application-key liitepyynto-key on-success on-error]}]
+    (ajax/GET (str "/lomake-editori/api/applications/" application-key
+                   "/field-deadline/" (name liitepyynto-key))
+              {:response-format (ajax/ring-response-format
+                                 {:format (ajax/json-response-format {:keywords? true})})
+               :handler         (fn [response]
+                                  (let [[date time last-modified] (parse-deadline-response response)]
+                                    (on-success date time last-modified)))
+               :error-handler   (fn [_] (on-error))
+               :headers         {"Caller-Id" (aget js/config "virkailija-caller-id")}})))
+
+(re-frame/reg-fx
+  :liitepyynto-information-request/save-deadline
+  (fn [{:keys [application-key liitepyynto-key deadline last-modified on-success on-error]}]
+    (ajax/PUT (str "/lomake-editori/api/applications/" application-key
+                   "/field-deadline/" (name liitepyynto-key))
+              {:response-format (ajax/ring-response-format
+                                 {:format (ajax/json-response-format {:keywords? true})})
+               :format          :json
+               :keywords?       true
+               :params          {:deadline (f/unparse iso-formatter deadline)}
+               :handler         (fn [response]
+                                  (let [[date time last-modified] (parse-deadline-response response)]
+                                    (on-success date time last-modified)))
+               :error-handler   (fn [_] (on-error))
+               :headers         (merge {"Caller-Id" (aget js/config "virkailija-caller-id")
+                                        "CSRF"      (cu/csrf-token)}
```

Comment:
Näitä yhteisiä headereita asetetaan nyt aika monessa paikassa. Jos niitä ei saa asetettua ajax-kirjastoon jotenkin globaalisti, pitäisikö ajax wrapata jollain omalla kilkkeellä, joka laittaisi nämä paikalleen?